### ISSUE TITLE
'GLIBC_2.28' not found running v1.0.0-Beta release

### ISSUE THREAD

**Issue by kingforaday** [DeimosC2/DeimosC2#7]
### Environment Data
* DeimosC2 Version: v1.0.0-Beta
* DeimosC2 Build: N/A
* Operating System: Ubuntu 18.04

ldd information:
```bash
user@deimosc2:~# ldd --version
ldd (Ubuntu GLIBC 2.27-3ubuntu1.2) 2.27
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

### Expected Behavior
N/A

### Actual Behavior
```bash
user@deimosc2:~# ./DeimosC2
./DeimosC2: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by ./DeimosC2)
```

### Steps to Reproduce Behavior
1) Use Ubuntu 18.04
2) Download release build v1.0.0-Beta

### Misc Information
Consider using an older build system for releases, example Debian 8 is glibc 2.19.

Build from source work fine from commit `0c6f0403d6964`
```bash
user@deimosc2:~/dev/DeimosC2/c2# ./c2
2020/07/27 15:27:54 main.go:132: C2 Server Starting...
2020/07/27 15:27:54 main.go:144: Main.go goroot:  /usr/local/go
2020/07/27 15:27:54 main.go:150: Main.go gopath:  /home/user/go
2020/07/27 15:27:54 main.go:161: It seems like this is your first time running DeimosC2.
2020/07/27 15:27:54 main.go:162: Would you like to restore from archive? [Y/N]
```

**Comment by paragonsec** [comment]
I will look into building on older versions. If we go this route we won't issue new releases until our next minor release which is planned. Thanks!

**Comment by paragonsec** [comment]
Fixed in the release 1.1.0-Beta.
